dataland-community-manager/src/main/kotlin/db/migration/V2__ExtendDataRequest.kt

```diff
@@ -0,0 +1,19 @@
+package db.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+
+/**
+ * This migration script creates an index for a faster search for company names and identifiers
```

Comment:
Is that the correct doc here?

Reply:
@fsoffel Probably not, could you adjust that or should I take it over?

Reply:
Btw, @fsoffel have you written a test for that? Do we need one?

Reply:
I don't know how to test that. However, I did not look into it as well. It was definitely manually tested multiple time, its quite independent from the rest of the code and also unlikely that someone ever touches it again. Thus a test is not a must IMO. We did not have tests for other simple SQL migrations so far as well.

Reply:
The docs are completely wrong :D I'm on it.

Reply:
Pushed

Reply:
Thank you and i think in this case here a test is not necessary.